webix_custom_components/formioBuilder.js

```diff
@@ -64,362 +68,334 @@ module.exports = class ABCustomFormIOBuilder {
-               switch (entry.field.key) {
-                  case "boolean":
-                     components[entry.key] = {
-                        title: entry.label,
-                        key: entry.key,
-                        icon: entry.field.icon,
-                        schema: {
-                           abFieldID: entry.field.id,
-                           label: entry.field.label,
-                           type: "checkbox",
-                           disabled: true,
-                           key: entry.key,
-                           _key: entry.key,
-                           input: true,
-                        },
-                     };
-                     break;
-                  case "calculate":
-                     components[entry.key] = {
-                        title: entry.label,
-                        key: entry.key,
-                        icon: entry.field.icon,
-                        schema: {
-                           abFieldID: entry.field.id,
-                           label: entry.field.label,
-                           type: "textfield",
-                           key: entry.key,
-                           _key: entry.key,
-                           input: true,
-                           inputType: "text",
-                           disabled: true,
-                           calculateValue:
-                              "value = " +
-                              entry.field.settings.formula
-                                 .replace(/{/g, "data['")
-                                 .replace(/}/g, "']"),
-                        },
-                     };
-                     break;
-                  case "connectObject":
-                     components[entry.key] = {
-                        title: entry.label,
-                        key: entry.key,
-                        icon: entry.field.icon,
-                        schema: {
-                           abFieldID: entry.field.id,
-                           label: entry.field.label,
-                           type: "textfield",
-                           key: entry.key,
-                           _key: entry.key,
-                           input: true,
-                           inputType: "text",
-                           disabled: true,
-                           calculateValue: `value = data['${entry.key}.format']`,
-                           // ,calculateValue: `value = '${entry.field.settings.textFormula}'`
-                        },
-                     };
-                     break;
+   /**
+    * @method parseDataObjects
+    *
+    * @param store {webix.TreeCollection}
+    * @param elem {Object} the webix element
+    * @param parentId {integer - nullable} id of parent id
+    */
+   parseDataFields(fields) {
+      const components = {};
+      // objects.forEach((obj) => {
+      //     var fields = obj.fields();
+      //     console.log(fields);
+      fields.forEach((entry) => {
```

Comment:
so what are these `fields` here?  are they instances of an `ABField` objects?  or are they another kind of json definition?  (I don't see us using any method on them)